pypdf/_writer.py

```diff
@@ -833,6 +833,19 @@ def _update_field_annotation(
                 ),
             ).get_object(),
         )
+
+        if "/Font" not in dr or not isinstance(dr["/Font"], DictionaryObject):
+            dr[NameObject("/Font")] = DictionaryObject()
+            font_dict = dr["/Font"]
+            font_entry = DictionaryObject({
+                NameObject("/Type"): NameObject("/Font"),
+                NameObject("/Subtype"): NameObject("/TrueType"),
+                NameObject("/BaseFont"): NameObject("/Helvetica"),
```

Comment:
Couldn't we make the font name configurable through a constant or method parameter?

Reply:
Helvetica is the default font that should be embedded in viewers